spec/services/idv/session_spec.rb

```diff
@@ -119,10 +135,17 @@
 
         it 'sets profile to pending in person verification' do
           subject.create_profile_from_applicant_with_password(user.password)
+          profile = subject.profile
 
           expect(subject).not_to have_received(:move_pii_to_user_session)
-          expect(subject.profile.active?).to eq(false)
-          expect(subject.profile.deactivation_reason).to eq('in_person_verification_pending')
+          expect(profile.activated_at).to eq nil
+          expect(profile.active).to eq false
+          expect(profile.deactivation_reason).to_not eq :in_person_verification_pending
```

Comment:
@soniaconnolly symbol fails, it's definitely a string

Reply:
That's surprising! Thanks for checking.